Source/WebKit/UIProcess/Extensions/WebExtensionSidebar.h

```diff
@@ -83,7 +83,7 @@ class WebExtensionSidebar : public API::ObjectImpl<API::Object::Type::WebExtensi
 
     /// `icon()` will return the overridden icon of this sidebar, or the icon of the first parent sidebar in which the icon is set
     RetainPtr<CocoaImage> icon(CGSize);
```

Comment:
Should also match `WebExtension` using `WebCore::Icon` and `WebCore::FloatSize`.